dist/obs-server.spec

```diff
@@ -876,7 +892,7 @@ usermod -a -G docker obsservicerun
 %{_unitdir}/obs-delayedjob-queue-releasetracking.service
 %{_unitdir}/obs-delayedjob-queue-staging.service
 %{_unitdir}/obs-sphinx.service
-%{_sbindir}/rcobs-api-support
+%if 0%{?suse_version}
 %{_sbindir}/rcobs-clockwork
```

Comment:
It seems `rcobs-api-support` got removed. Any reason?

Reply:
This was previously discussed in the original PR, but it doesn't work because the `rcFOO` stuff only works with `.service` files. It's not supposed to be used with `.target` and `.timer` files.

Reply:
Uhh,  I have to disagree. Basically it seems to work also for targets:

```
nextcloud:~ # rcredis status
* redis.target - Redis target allowing to start/stop all redis@.service instances at once
```

Reply:
It shouldn't... But I guess I can put it back since it clearly seems to...

Reply:
Restored.